recipe/meta.yaml

```diff
@@ -50,13 +53,26 @@ requirements:
 
 test:
   requires:
-    - pytest >=5.0.1
     - cython >=0.29.24
-    - pytest-xdist
-    - pytest-timeout
+    - llvm-openmp    # [osx]
```

Comment:
I would leave out llvm-openmp has it's already set in the run section.

Reply:
Thanks. I've removed it